webpack/scenes/Settings/Tables/__tests__/TableActions.test.js

```diff
@@ -0,0 +1,84 @@
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
+import Immutable from 'seamless-immutable';
+import configureMockStore from 'redux-mock-store';
+import { mockRequest, mockReset } from '../../../../mockRequest';
+import { loadTables, createColumns, updateColumns } from '../TableActions';
+import {
+  requestSuccessResponse,
+  getSuccessActions,
+  getFailureActions,
+  createSuccessActions,
+  createFailureActions,
+  updateSuccessActions,
+  updateFailureActions,
+  tableRecord,
+} from './Table.fixtures';
+
+const accessDenied = {
+  error: {
+    message: 'Access denied',
+    details: 'You are trying access the preferences of a different user',
+  },
+};
+const mockStore = configureMockStore([thunk]);
+const store = mockStore({ settings: Immutable({}) });
+const testTableName = 'Katello::Subscriptions';
+
+beforeEach(() => {
+  store.clearActions();
+  mockReset();
+});
+
+describe('table actions', () => {
```

Comment:
You could write much simpler test here if you would use the test-helper `testActionSnapshotWithFixtures`.
https://github.com/Katello/katello/blob/master/webpack/move_to_pf/test-utils/testHelpers.js#L58

We still don't have an example in `katello` but you can see an example in `foreman`:
https://github.com/theforeman/foreman/blob/fe74cc88d888cbf51f441c253e027b007440e77f/webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/BreadcrumbBarActions.test.js

If you want to, it could be nice to use the `testActionSnapshotWithFixtures`.

Reply:
And you would be able to remove bunch of fixtures because it will create snapshots instead.